lib/seek/util.rb

```diff
@@ -180,5 +180,11 @@ def self.filter_disabled(types)
       types.select(&:feature_enabled?)
     end
 
+    def self.extended_metadata_supported_types
+      ActiveRecord::Base.descendants.select do |model|
+        model.respond_to?(:supports_extended_metadata?) && model.supports_extended_metadata?
+      end.map(&:name)
```

Comment:
return the types, instead of their String names - to be consistent with the other Util methods

Reply:
by which I mean the classes